authors/apps/authentication/urls.py

```diff
@@ -1,15 +1,18 @@
-from django.conf.urls import url
 from rest_framework_swagger.views import get_swagger_view
+from django.urls import path
+from django.conf.urls import url
 
 from .views import (
-    LoginAPIView, RegistrationAPIView, UserRetrieveUpdateAPIView
+    LoginAPIView, RegistrationAPIView, UserRetrieveUpdateAPIView, ActivationView
 )
 
 swagger_view = get_swagger_view(title='The Jedi Authors Haven API')
 
 urlpatterns = [
-    url(r'^user/?$', UserRetrieveUpdateAPIView.as_view()),
-    url(r'^users/?$', RegistrationAPIView.as_view()),
-    url(r'^users/login/?$', LoginAPIView.as_view()),
+    path('user/', UserRetrieveUpdateAPIView.as_view()),
```

Comment:
Looks good to me. Nice job in implementing the use of `path` instead of `url` in the application.